components/ui/Drawer.tsx

```diff
@@ -53,10 +53,10 @@ function Drawer(props: Props) {
         {children}
       </div>
 
-      <aside class="drawer-side h-full z-50">
+      <div class="drawer-side h-full z-50" style={{ overflow: "hidden" }}>
```

Comment:
Doesn't this work?

```suggestion
      <div class="drawer-side h-full z-50 overflow-hidden">
```

Reply:
Why need change from aside to div? The aside has a meaning/semantic. You can edit the aside css to be equal to the div.

Reply:
I just test with div. I didn't test with aside, so maybe just trade the overflow is enough